### PR TITLE
#62  add new design to the app

### PR DESCRIPTION
**Pull request by mavv-coder**
Create session page styles updated for now

### CODE REVIEW

File: front/src/common-app/components/appbar/appbar.styles.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  margin-left: auto;
+  list-style: none;
+`;
+
+export const listItem = css`
+  margin-right: 1.5rem;
+  overflow: hidden;
+
+  &:last-of-type {
+    margin-right: 2.4rem;
```
Comment (brauliodiez) [inline front/src/common-app/components/appbar/appbar.styles.ts:37]: Not sure if we should calculate this based on the standard Theme spacing measure adding @Nasdan to the loop

File: front/src/common-app/components/footer/footer.component.tsx [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+export const GithubIcon: React.FC<Props> = ({ className }) => {
+  return (
+    <svg className={className} aria-hidden="true" viewBox="0 0 2350 2314.7">
+      <path d="M1175,0C525.8,0,0,525.8,0,1175c0,552.2,378.9,1010.5,890.1,1139.7-5.9-14.7-8.8-35.3-8.8-55.8V2059.1H734.4c-79.3,0-152.8-35.2-185.1-99.9-38.2-70.5-44.1-179.2-141-246.8-29.4-23.5-5.9-47,26.4-44.1,61.7,17.6,111.6,58.8,158.6,120.4s67.6,76.4,155.7,76.4c41.1,0,105.7-2.9,164.5-11.8,32.3-82.3,88.1-155.7,155.7-190.9-393.6-47-581.6-240.9-581.6-505.3,0-114.6,49.9-223.3,132.2-317.3-26.4-91.1-61.7-279.1,11.8-352.5,176.3,0,282,114.6,308.4,143.9,88.1-29.4,185.1-47,284.9-47,102.8,0,196.8,17.6,284.9,47,26.4-29.4,132.2-143.9,308.4-143.9,70.5,70.5,38.2,261.4,8.8,352.5,82.3,91.1,129.3,202.7,129.3,317.3,0,264.4-185.1,458.3-575.7,499.4,108.7,55.8,185.1,214.4,185.1,331.9V2256c0,8.8-2.9,17.6-2.9,26.4C2021,2123.8,2350,1689.1,2350,1175,2350,525.8,1824.2,0,1175,0Z" />
+    </svg>
+  );
+};
+
+// Lemoncode logo svg - - - - -
+export const LemoncodeIcon: React.FC<Props> = ({ className }) => {
```
Comment (brauliodiez) [inline front/src/common-app/components/footer/footer.component.tsx:87]: Maybe we can include the svg file an import it? Did it gave you an issue? I remember @Nasdan told me some trick about SVG's?

File: front/src/common-app/components/appbar/appbar.styles.ts (hunk 1 above)
Comment (Nasdan) [reply front/src/common-app/components/appbar/appbar.styles.ts:37]: Yes it's. You could use `theme.spacing(<number>)` where number could be `1` or `1.5` or `-2.445`...

The rem units are similar to me, but it depends on font-size's html element

File: front/src/common-app/components/footer/footer.component.tsx (hunk 2 above)
Comment (Nasdan) [reply front/src/common-app/components/footer/footer.component.tsx:87]: If you want to create the react component from svg file, it exists a library [@svgr/webpack](https://github.com/gregberge/svgr/tree/master/packages/webpack) where you can configure a webpack loader to convert this kind of files  and then import it as React components:

image

**Pull request merged**